src/pytest_redis/executor.py

```diff
@@ -115,6 +119,9 @@ def __init__(
         self.unixsocket = str(datadir / f"redis.{port}.sock")
         self.executable = executable
 
+        self.username = username
```

Comment:
username isn't used for RedisExecutor

Reply:
yes i set it for consistency reasons so all executors have a username/password attribute.